projects/demo/src/modules/app/app.routes.ts

```diff
@@ -777,6 +777,16 @@ export const ROUTES = [
             title: 'Slider',
         },
     },
+    {
+        path: 'components/slider-old',
```

Comment:
@waterplea @nsbarsukov What do you think about path as `slider[legacy]` or `slider[deprecated]`instead of `slider-old`

Reply:
I don't really care because it's going away in 3.0 anyway

Reply:
I have changed route path to `slider[deprecated]`